DataFormats/JetReco/src/classes_3.h

```diff
@@ -5,6 +5,12 @@
 
 #include "DataFormats/JetReco/interface/BasicJetCollection.h"
 #include "DataFormats/JetReco/interface/CaloJetCollection.h"
+#if defined __has_feature
+#if __has_feature(modules)
+// Workaround the missing CLHEP.modulemap: CLHEP/Vector/ThreeVector.h:41:7: error: redefinition of 'Hep3Vector'
+#include "DataFormats/JetReco/interface/PFClusterJetCollection.h"
```

Comment:
similar to #28925 , I was expecting that DataFormats/JetReco/interface/PFClusterJetCollection.h (already mentioned below) is paired up with ```#include "CLHEP/Vector/ThreeVector.h"```

Otherwise, what is the benefit of this conditional double-inclusion.
Can this include be here just unconditionally or (still unconditionally) twice, if double-mention is needed?

Reply:
Good questions.

I think PR #28925 may have been good enough with the single include without the `ThreeVector.h`. The include order matters because we do not yet have a module for CLHEP. This means that CLHEP headers will be contained in other modules while some includes (which include CLHEP headers) will parse.

Moving the include up activates the copy of the CLHEP headers from `PFClusterJetCollection.h` rather than reparsing and thus avoiding a bug in rootcling (clang).

Reply:
> PS: I would rather keep the duplicate include so that once we have the CLHEP module I will remove the workarounds altogether.

did you mean to keep it unconditionally (change this PR), or stay with the conditional duplication?
Please clarify.

Reply:
I meant to keep it conditional as it is now in the PR.